libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -101,21 +102,35 @@ std::string YulUtilFunctions::copyToMemoryFunction(bool _fromCalldata, bool _cle
 			("cleanup", _cleanup)
 			.render();
 		}
+		// memory -> memory
 		else
 		{
-			return Whiskers(R"(
-				function <functionName>(src, dst, length) {
-					let i := 0
-					for { } lt(i, length) { i := add(i, 32) }
-					{
-						mstore(add(dst, i), mload(add(src, i)))
+			// use MCOPY (cancun+)
```

Comment:
This comment and the ones above seem pretty redundant to me... if a comment is an additional line explaining pretty much what the next line says implicitly anyways, maybe we don't need a comment...